src/reducers/targetsReducer.js

```diff
@@ -1,9 +1,29 @@
 import { createReducer } from '@rootstrap/redux-tools';
-import { getTargetsSuccess } from 'actions/targetActions';
+import {
+  createTargetReset,
+  createTargetSuccess,
+  getTargetsSuccess,
+} from 'actions/targetActions';
 
 const initialState = { targetsList: [] };
 
 const targetsReducer = {
+  [createTargetReset]: (store, action) => {
+    store.matchedUser = undefined;
+    store.matchId = undefined;
+  },
+
+  [createTargetSuccess]: (store, action) => {
+    if (action.payload.matchConversation) {
```

Comment:
destructure the action outside the if so you can use it here
```suggestion
    if (matchConversation) {
```